seed/static/seed/js/controllers/inventory_detail_analyses_controller.js

```diff
@@ -103,6 +103,19 @@ angular.module('BE.seed.controller.inventory_detail_analyses', [])
         })
       }
 
+      $scope.inventory_display_name = function(property_type) {
+        let error = '';
+        let field = property_type == "property" ? $scope.org.property_display_field : $scope.org.taxlot_display_field;
+        if (!(field in $scope.item_state)) {
+          error = field + ' does not exist';
+          field = 'address_line_1';
+        }
+        if (!$scope.item_state[field]) {
```

Comment:
Looks like I'm getting both messages:
<img width="494" alt="Screen Shot 2021-01-29 at 1 51 58 PM" src="https://user-images.githubusercontent.com/18518728/106327820-fea8cd00-623b-11eb-8f90-44b6d98cde62.png">

Reply:
Modified to accomodate.